### PR TITLE
[2506] LuNumericBadge : Support string type

### PR DESCRIPTION
**Pull request by g-drouard**
## Description

LuNumericBadge: Add `string` type

-----

-----

### CODE REVIEW

**Comment by vvalentin-lucca** [comment]
Wouldn't it have been simpler and more flexible to authorize a text type?

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11718/ui-diff)

**Comment by g-drouard** [comment]
> Wouldn't it have been simpler and more flexible to authorize a text type?

I agree but in this case does the name of the component still make sense?

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2508/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11756/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2508/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11757/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2508/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11762/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2508/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11887/ui-diff)

**Review by Supamiu — APPROVED** [review]

File: packages/ng/numeric-badge/numeric-badge.component.ts [hunk 1]
```diff
@@ -16,11 +16,11 @@ import { NgClazz, Palette } from '@lucca-front/ng/core';
 export class NumericBadgeComponent implements OnChanges {
 	#ngClass = inject(NgClazz);
 
-	@Input({ required: true, transform: numberAttribute })
+	@Input({ required: true })
```
Comment (shajz) [inline packages/ng/numeric-badge/numeric-badge.component.ts:19]: I don't understand this change, this is a component made to display numbers, the `numberAttribute` seems like a perfect fit. https://angular.io/api/core/numberAttribute

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2508/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11917/ui-diff)

File: packages/ng/numeric-badge/numeric-badge.component.ts (hunk 1 above)
Comment (shajz) [reply packages/ng/numeric-badge/numeric-badge.component.ts:19]: Alright ! Maybe these examples could be integrated in the story to give a better understanding of the favored use cases ?

Comment (Supamiu) [reply packages/ng/numeric-badge/numeric-badge.component.ts:19]: Ping @jeremie-lucca

Comment (jeremie-lucca) [reply packages/ng/numeric-badge/numeric-badge.component.ts:19]: Missed that one, sorry! 
Indeed, we have multiple feedback that requires to add extra characters which lead UI devs to switch to HTML component.
And agree, I'll add extra documentation on Prisme.

**Review by jeremie-lucca — APPROVED** [review]

**Pull request merged**